lib/restaurant.rb

```diff
@@ -0,0 +1,26 @@
+require './lib/menu.rb'
+require './lib/order.rb'
+require './lib/twilio.rb'
+
+class Restaurant
+
+  attr_reader :order
+
+  def initialize(order = Order.new)
+    @order = order
+  end
+
+  def total_correct(total)
+    total == order.total
+  end
+
+  def place_order(list=basket_list, total)
```

Comment:
Style/OptionalArguments: Optional arguments should appear at the end of the argument list. (https://github.com/bbatsov/ruby-style-guide#optional-arguments)